kits/scale-pilot/prompts/scale-pilot_instructor-llmnode-493_user_1.md

```
@@ -0,0 +1,4 @@
+Analyze the user's software architecture description and extract the information into the defined JSON schema.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)